projects/buf.build/package.yml

```diff
@@ -0,0 +1,33 @@
+distributable:
+  url: https://github.com/bufbuild/buf/archive/refs/tags/v{{version}}.tar.gz
+  strip-components: 1
+
+versions:
+  github: bufbuild/buf
+
+provides:
+  - bin/buf
+
+build:
+  script:
+    - make build install
+    - mkdir -p "{{ prefix }}"/bin
+    - mv ~/.cache/buf/$PLATFORM/gobin/buf "{{ prefix }}"/bin
```

Comment:
We should really try to avoid letting this pollute a user's $HOME. https://buf.build/docs/installation#from-source only talks about building from github sources, not local sources.

I suspect our logic should look something like https://github.com/teaxyz/pantry/blob/main/projects/gitlab.com/gitlab-org/cli/package.yml.

Reply:
Thanks for the feedback, @jhheider!

Reply:
No problem!